src/components/ReviewCards.jsx

```diff
@@ -0,0 +1,24 @@
+const ReviewCards = ({ reviews }) => {
+    return (
+        <ul id="reviewList">
+            {reviews.map((review) => {
+                return (
+                    <li key={review.review_id} class="reviewCards">

```

Comment:
With React, class should be className. Just make sure to change these in your components.